test/factory/common_factory_tests.dart

```diff
@@ -64,41 +66,41 @@ void commonFactoryTests(ReactComponentFactoryProxy factory, {bool isFunctionComp
 
         test('$childrenCount', () {
           final expectedChildren = new List.generate(childrenCount, (i) => i + 1);
-          final arguments = <dynamic>[{...props}]..add(expectedChildren);
+          final arguments = <dynamic>[props, ...expectedChildren];
```

Comment:
Uhh this was actually an bug in the test I didn't notice until I went to convert it to use ui-as-code syntax; it should have been `..addAll(expectedChildren)`